apps/core-rollup-js/src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Clarity Design System</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="Demo of Clarity Core Performance" />
+    <link rel="stylesheet" href="/index.css" />
+  </head>
+  <body cds-text="body">
+    <main cds-layout="vertical gap:lg p:lg p-t:xl container:xs container:center">
+      <h1 cds-text="heading">Clarity Design <cds-badge status="success"></cds-badge></h1>
```

Comment:
What are your thoughts about loading an icon and displaying it on this page somewhere? That seems like something that people will want to be able to do and there are a couple more steps involved as compared to the badge.

Reply:
This will also be good as a demo showing the a isolated demo with icon treeshaking